myInsta/myInsta/MIPost.h

```diff
@@ -0,0 +1,28 @@
+//
+//  Post.h
+//  myInsta
+//
+//  Created by Abigail Shilts on 6/28/22.
+//
+
+#import <Foundation/Foundation.h>
+#import <Parse/Parse.h>
+#import "Parse/Parse.h"
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MIPost : PFObject<PFSubclassing>
+
+@property (nonatomic, strong) NSString *postID;
+@property (nonatomic, strong) NSString *userID;
+@property (nonatomic, strong) NSString *caption;
+@property (nonatomic, strong) NSNumber *likeCount;
+@property (nonatomic, strong) NSNumber *commentCount;
+@property (nonatomic, strong) PFFileObject *image;
+@property (nonatomic, strong) PFUser *author;
+
++ (void) postUserImage: ( UIImage * _Nullable )image withCaption:
```

Comment:
This API looks odd, all the parameters are optional. Is it because you can post having just image or caption?